src/test/java/bio/terra/pdao/bigquery/BigQueryPdaoTest.java

```diff
@@ -23,12 +42,34 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
-@Category(Connected.class)
+@AutoConfigureMockMvc
 @ActiveProfiles("google")
+@Category(Connected.class)
 public class BigQueryPdaoTest {
 
-    @Autowired
-    private BigQueryPdao bigQueryPdao;
+    @Autowired private MockMvc mvc;
+    @Autowired private ObjectMapper objectMapper;
+    @Autowired private JsonLoader jsonLoader;
+    @Autowired private DataRepoConfiguration dataRepoConfiguration;
+    @Autowired private Storage storage;
+    @Autowired private BigQueryPdao bigQueryPdao;
+
+    @MockBean
+    private SamClientService samService;
+
+    private ConnectedOperations connectedOperations;
+
+    @Before
+    public void setup() throws Exception {
+        // Setup mock sam service
+        ConnectedOperations.stubOutSamCalls(samService);
+        connectedOperations = new ConnectedOperations(mvc, objectMapper, jsonLoader);
+    }
```

Comment:
maybe we should have a connected test base class that does the above 2 things so that we aren't importing sam stuff for tests that shouldn't need it. (again, Dan M you don't need to make this refactor)